README.md

```diff
@@ -57,6 +57,8 @@ If you turn the [Doom logo upside down](https://www.reddit.com/r/Doom/comments/8
  * In-game music changing using MUSINFO.
  * UMAPINFO support, compliant to Rev 2.2 of the [spec](https://github.com/kraflab/umapinfo).
  * MBF21 compatibility level, compliant to Rev 1.4 of the [spec](https://github.com/kraflab/mbf21).
+ * Partial support of the ID24 [spec](https://drive.google.com/drive/folders/1okmh2M2pIjIKnkJNsdphSWrmtp0_MncU?usp=sharing) (interlevel animation, SKYDEFS, SBADEF).
+ * Support for PNG graphics. Partial support of resource directories and PK3 archives.
```

Comment:
Aren't these two separate points?

Reply:
Honestly, I don't like mentioning "partial" support. I think we should mention PK3 when we add the "textures" subfolder, and ID24 when we decide what to implement and what not.

Reply:
Alright, fine with me.